src/Microsoft.ApplicationInsights.AspNetCore/Implementation/TelemetryConfigurationOptionsSetup.cs

```diff
@@ -45,6 +49,15 @@ public void Configure(TelemetryConfiguration configuration)
                 configuration.InstrumentationKey = this.applicationInsightsServiceOptions.InstrumentationKey;
             }
 
+            if (this.telemetryProcessorFactories.Any())
```

Comment:
Actually, I wonder if we should remove the if statement altogether and let it fall through to the (potentially empty) ```foreach``` followed by ```.Build()```. In the case where users have manually added to the ```TelemetryProcessorChainBuilder```, someone needs to call ```.Build()``` and this might be the most appropriate place.